pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -49,6 +48,14 @@ def on_trainer_init(
         # pass through the required args to figure out defaults
         self.configure_checkpoint_callbacks(checkpoint_callback)
 
+        # Set refresh rate to 20 on colab
+        # to prevent crashing because of the progress bar
+        if IS_COLAB and progress_bar_refresh_rate < 20:
+            rank_zero_warn(
+                "You have set progress bar refresh rate to less than 20 on Google Colab. This "
+                " may cause crashes. Consider using Trainer(progress_bar_refresh_rate=20...)",
```

Comment:
```suggestion
                "You have set progress_bar_refresh_rate < 20 on Google Colab. This"
                " may crash. Consider using progress_bar_refresh_rate>=20 in Trainer.",
```

Reply:
should this warning be there in `configure_progress_bar`?

Reply:
Yes, that would be better